package.json

```diff
@@ -20,6 +20,7 @@
 		"build:full:compile": "fluid-build --task compile --task webpack",
 		"build:gendocs": "concurrently build:gendocs:* && cd docs && npm run build",
 		"build:gendocs:azure": "copyfiles azure/packages/*/_api-extractor-temp/** ./_api-extractor-temp/doc-models/ -f -V",
+		"build:gendocs:packages": "copyfiles packages/**/_api-extractor-temp/** ./_api-extractor-temp/doc-models/ -f -V",
```

Comment:
Since packages now save their `.api.json` metadata reports in their own directories, this ensures all of the data necessary for the website build artifacts is hoisted to the root as it was before.